### PR TITLE
initial configuration, largely based on TF, but without UI configs fo…

### PR DESCRIPTION
**Pull request by fillup**
left out UI config for now while we test Cloudflare Pages. I don't know if we'll really need all those vars for the app, but they seemed likely based on TF

### CODE REVIEW

File: main-api.tf [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+  protocol             = var.disable_tls == "true" ? "HTTP" : "HTTPS"
+  vpc_id               = data.terraform_remote_state.common.outputs.vpc_id
+  deregistration_delay = "30"
+
+  stickiness {
+    type = "lb_cookie"
+  }
+
+  health_check {
+    path     = "/"
```
Comment (Schparky) [inline main-api.tf:31]:
```suggestion
    path     = "/site/status"
```

File: main-api.tf [hunk 2]
```diff
@@ -0,0 +1,317 @@
+/*
+ * Create ECR repo
+ */
+module "ecr" {
+  source              = "github.com/silinternational/terraform-modules//aws/ecr?ref=3.6.2"
+  repo_name           = "${var.app_name}-${data.terraform_remote_state.common.outputs.app_env}"
```
Comment (Schparky) [inline main-api.tf:6]: If you like, you can make a local for this:
```
locals {
    app_name_and_env = "${var.app_name}-${data.terraform_remote_state.common.outputs.app_env}"
}
```
and use it as `local.app_name_and_env`

**Review by Schparky — APPROVED** [review]

File: main-api.tf (hunk 2 above)
[the remark this answers is not in the document]
Comment (fillup) [reply main-api.tf:6]: good idea

Comment (fillup) [reply main-api.tf]: updated

[the remark this answers is not in the document]
Comment (fillup) [reply main-api.tf]: updated

**Pull request merged**